easy_phi/app.py

```diff
@@ -261,6 +274,86 @@ def get(self):
                 )
             )
 
+class WebSocketHandler(tornado.websocket.WebSocketHandler):
+    allow_broadcast = False
+
+    def update_module(self, added, slot):
+        err = ''
+        max_slot = len(hwconf.modules) - 1
+        min_slot = 0 if self.allow_broadcast else 1
+        if not min_slot <= slot <= max_slot:  # invalid slot number
+            err = 'Invalid slot number. Number in range ' + \
+                  '{0}..{1} expected'.format(min_slot, max_slot)
+        if err:
+            message = {
+                'msg_type': 'ERROR',
+                'message': err
+            }
+        else:
+            message = {
+                'msg_type': 'MODULE_UPDATE',
+                'slot': slot,
+                'module_name': getattr(hwconf.modules[slot], 'name', None),
+                'added': added
+            }
+        self.write_message(message)
+
+    def update_lock(self, slot, used_by):
+        err = ''
```

Comment:
besides this check looks unnecessary, it is repeated three times. If you have a repeated piece of code, usually it is a reason to move it to function